src/index.html

```diff
@@ -15,6 +15,14 @@
       body.empty {
         background-color: rgb(50,50,50);
       }
+      body.dark {
+        background-color:  #272525;
+      }
+      #output.dark, #output.dark td, #output.dark th, #output.dark pre {
+        background-color:  #272525;
+        color: #f1f1f1;
+        border-color: #f1f1f1;
+      }
```

Comment:
I think now we can remove the `isolation: isolate;` from line 39 and the `class="darkmode-ignore"` from line 87.